provisioner/terraform/testdata/calling-module/calling-module.tfplan.json

```diff
@@ -259,7 +259,7 @@
       ]
     }
   ],
-  "timestamp": "2025-01-28T15:12:27Z",
+  "timestamp": "2025-01-29T22:47:46Z",
```

Comment:
(not in scope of this PR): I wonder if we can replace these with placeholders? It's not really useful to have these change every time they're regenerated.